boards/arm/efr32bg_sltb010a/efr32bg_sltb010a.dts

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright (c) 2021 Sateesh Kotapati
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <silabs/efr32bg22c224f512im40.dtsi>
+#include <silabs/efr32bg22-pinctrl.dtsi>
+
+/ {
+	model = "Silicon Labs EFR32BG SLTB010A (aka Thunderboard BG22)";
+	compatible = "silabs,efr32bg_sltb010a", "silabs,efr32bg22";
+
+	/* These aliases are provided for compatibility with samples */
+	aliases {
+		led0 = &led0;
+		sw0 = &button0;
+	};
+
+	chosen {
+		zephyr,console = &usart1;
+		zephyr,shell-uart = &usart1;
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		led0: led_0 {
+			gpios = <&gpiob GECKO_PIN(0) GPIO_ACTIVE_HIGH>;
+			label = "LED 0";
+		};
+	};
+
+	buttons {
+		compatible = "gpio-keys";
+		button0: button_0 {
+			gpios = <&gpiob GECKO_PIN(1) GPIO_ACTIVE_LOW>;
+			label = "User Push Button 0";
+		};
+	};
+
+	zephyr,user {
```

Comment:
NAK. The `/zephyr,user` node is reserved for application specific use; you cannot use it in board files. This needs to be removed wherever it appears outside of applications in thi sPR.


 One option for reworking this in board files is to use a node at a fixed location and use DT_PATH to get the node identifier in your C code.

Reply:
note we use it now in a few boards, I guess we need to use board level compatibles then?

Reply:
For now, I left the old `gpio-keys` compatible and renamed the node according to its function. I will prepare board level compatible for that and I will update the PR soon.

Reply:
>  I guess we need to use board level compatibles then?

yes

Reply:
I changed this to use board level compatible so it should be OK now.